DirectX11Proj/modelclass.cpp

```diff
@@ -1,8 +1,11 @@
 #include "modelclass.h"
 
 #include "d3dVertexBuffer.h"
+#include "d3dclass.h"
 #include "ModelData.h"
```

Comment:
I use camel case for everything, d3dclass is partially legacy. renaming it is possible